Harmony/Classes/Future/Observable+Time.swift

```diff
@@ -25,9 +25,9 @@ public extension Observable {
     ///   - queue: The queue to schedule the delay (by default the Main Queue).
     /// - Returns: A chained observable.
     func withDelay(_ interval: TimeInterval, queue: DispatchQueue = DispatchQueue.main) -> Observable<T> {
```

Comment:
Having the `DispatchQueue.main` as the default queue is dangerous as it could be used in a datasource, repository, etc... and the business logic it could be executed in the main thread.

Reply:
This is true, it is a risk. The idea of this default value was to give a user-friendly usage on async delays. However, we could remove the default value to ensure programmers are using the correct queue instead of a default one. Will update this.

Reply:
👍